lib/coverband/adapters/redis_store.rb

```diff
@@ -11,7 +11,7 @@ class RedisStore < Base
       # used to store data to redis. It is changed only when breaking changes to our
       # redis format are required.
       ###
-      REDIS_STORAGE_FORMAT_VERSION = 'coverband3_1'
+      REDIS_STORAGE_FORMAT_VERSION = 'coverband_3_2'
```

Comment:
I know we talked about this before but this is going to be confusing for us to keep track of in our brains. We do always have git but we might want to think about just making this match the coverband version number whenever we bump this format version.

Reply:
I think it is worth keeping it independent so we can change the version differently than our gem version. We renamed the constant to improve that confusion. I think we could try to change the numbering significantly to help as well. If we are going to introduce the concept of migrations and such I really don't want it tied at all to Coverband version as this is far more stable.

I am starting to think about it as a schema_version, we could use a timestamp to match Rails schema but the timestamp wouldn't be user generate but updated when we are forcing a version change?

Reply:
I like the idea of modeling after rails migrations and makes sense that we would manually update it as this doesn't happen too often.

Reply:
I think it exactly serves its purpose here. If the data structure changes in a non-compatible way, this is the way to manage it without creating incompatibility.

Of course, if people want to keep their data on upgrade, they can write their own migrations, or the project can decide it's worthwhile providing those migrations for them. I think that in the majority of cases, the data is by-and-large disposable. (of course, people want to keep it, but telling them "this upgrade will wipe your coverage data, unless you do X" seems reasonable to me).

Reply:
What I like about the migration rake task with the `REDIS_STORAGE_FORMAT_VERSION` change is that it doesn't really wipe the data. It simply starts storing the data in a different spot. A likely scenario is that people will upgrade coverband and go to view the report and see their data is gone. If they care, they will either go to our changelog or README and see instructions there or open an issue on github and we can instruct them. In the end, they can recover their data if they want to.

Reply:
Leaving stale data behind isn't great either, though. But I agree with you...

Reply:
True but I think coverband doesn't take up too much space and the default ttl should clean things up eventually.

Reply:
Also, rake migrations can wipe old data after the update.